python/cugraph/centrality/betweenness_centrality_wrapper.pyx

```diff
@@ -26,82 +27,227 @@ import cudf
 import numpy as np
 import numpy.ctypeslib as ctypeslib
 
+import dask_cudf
+import dask_cuda
+import cugraph.raft
```

Comment:
The effort is to encapsulate raft in cugraph Comms. A raft import here can be avoided

Reply:
Updated in 09cbb8c